### ISSUE TITLE
findDOMNode is deprecated in StrictMode

### ISSUE THREAD

**Issue by yanai101** [primefaces/primereact#1404]
this is the error message that I have  in the console: (using ts )
```
Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of Transition which is inside StrictMode. Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here: https://fb.me/react-strict-mode-find-node
    in div (created by GrowlMessage)
    in GrowlMessage (created by Growl)
    in Transition (created by CSSTransition)
    in CSSTransition (created by Growl)
...
```

this is a code sandbox:
https://codesandbox.io/s/vibrant-haze-pxqw6

the error is trigger when the Message/ Growl component  change is inside state ...

Thanks

**Comment by ckay2003** [comment]
Seems to happen in multiple components. I get this message when using a calendar as a custom editor in datatable

**Comment by mertsincan** [comment]
Hi,

It is a warning, not an error. Also, this issue is related to transition API. Please see; https://github.com/reactjs/react-transition-group/issues/429

Also, https://github.com/facebook/react/issues/9217#issuecomment-292159323

**Comment by mertsincan** [comment]
Hi,

Fixed in #1790

Best Regards,
